server/requirements.txt

```diff
@@ -26,7 +26,7 @@ hf-transfer==0.1.6 ; python_version >= "3.9" and python_version < "3.13"
 huggingface-hub==0.22.2 ; python_version >= "3.9" and python_version < "3.13"
 humanfriendly==10.0 ; python_version >= "3.9" and python_version < "3.13"
 idna==3.7 ; python_version >= "3.9" and python_version < "3.13"
-importlib-metadata==7.2.1 ; python_version >= "3.9" and python_version < "3.13"
+importlib-metadata~=6.0.0 ; python_version >= "3.9" and python_version < "3.13"
```

Comment:
Any reason for the downgrade?

Reply:
It conflicted with the open-telemetry dependencies. I have promoted the open-telemetry versions more, and they look fine now.